.github/workflows/ci_tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           profile: minimal
           toolchain: stable
-      - uses: Swatinem/rust-cache@v2.0.0
+      - uses: Swatinem/rust-cache@v2.4.0
```

Comment:
How come this was updated?

Reply:
I just applied the open PR from dependabot